tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_controller_component.ts

```diff
@@ -194,4 +194,31 @@ export class LinkedTimeFobControllerComponent {
     // No fobs can pass the lowest step in this graph.
     return 0;
   }
+
+  stepTyped(fob: Fob, step: number) {
+    let newLinkedTime = this.linkedTime;
+    if (!newLinkedTime.end) {
```

Comment:
Note that !newLinkedTime.end will evalute to true when end is 0. Is that intentional? Either way, mind adding a test for that case?

Reply:
handled in summary comment.